plugins/workbench-impl/src/png.d.ts

```diff
@@ -0,0 +1 @@
+declare module '*.png'
```

Comment:
Why we need it?

Reply:
svelte-check is not passed if png files are imported without this kind of definition.